### ISSUE TITLE
[Bug]: Album art background tearing

### ISSUE THREAD

**Issue by TheGeeKing** [daksh2k/Spicetify-stuff#93]
### 🔍 Have you checked that you are using an auto updating method or latest version of the extension?

- [X] I have checked that I am on the latest version.

### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Choose the extension for bug report.

Full Screen Mode

### ℹ Environment / Computer Info

```Markdown
- Spotify version: Spotify for desktop-about.platform-win-x86-64 1.2.14.1149.ga3ae422d
- Spicetify version: Spicetify v2.20.1
```

### 📝 Description

Opening the Default Mode with the bottom left shortcut or F key makes the album art background being teared vertically at always the same spot with every songs. It fixes itself when it goes to the next song.

### 📸 Screenshots

![image](https://github.com/daksh2k/Spicetify-stuff/assets/58857539/d05cd9c1-603b-466a-aab2-e10c56048c7c)

**Comment by daksh2k** [comment]
That's weird, I faced this issue before fixing the canvas logic. Could you let me know if you're using the latest version?

If yes, can you share more details for reproducing this, like screen size, does it happen for specific settings, and your device specs (OS, Cpu, GPU etc.) , some scaling issue might be there which might be device-specific, as no other user reports are there similar to this.

**Comment by daksh2k** [comment]
@TheGeeKing closing this, as this does not seem to be an issue anymore. Can't reproduce this issue.

**Comment by TheGeeKing** [comment]
Still occuring

```
- Spotify version: Spotify for Windows (64 bits) 1.2.16.947.gcfbaa410
- Spicetify version: Spicetify v2.21.0
```

![image](https://github.com/daksh2k/Spicetify-stuff/assets/58857539/b70f4f2c-fc18-4ef1-a455-597de40fff43)

Here is a video if it can help
https://github.com/daksh2k/Spicetify-stuff/assets/58857539/f9c027e5-8aa6-464c-8dab-c39c2cffb8e5

**Comment by daksh2k** [comment]
Got the issue, thanks for the video 👍 , that really helped. 
It happens when the spotify window is small and goes into fullscreen when opening the extension. Will look into it.

**Comment by daksh2k** [comment]
Should be fixed in `v2.3.1`
